src/langchain_google_alloydb_pg/indexes.py

```diff
@@ -28,12 +28,16 @@ class StrategyMixin:
 class DistanceStrategy(StrategyMixin, enum.Enum):
     """Enumerator of the Distance strategies."""
 
-    EUCLIDEAN = "<->", "l2_distance", "vector_l2_ops"
-    COSINE_DISTANCE = "<=>", "cosine_distance", "vector_cosine_ops"
-    INNER_PRODUCT = "<#>", "ip_distance", "vector_ip_ops"
+    PGVECTOR_EUCLIDEAN = "<->", "l2_distance", "vector_l2_ops"
```

Comment:
This design makes it harder for our customers. They shouldn't need to care if its PGVector or Scann, they just need a distance strategy. It also doesn't work with our BaseIndex with default distance strategy.

Reply:
I have updated the distance strategy design to add an attribute `scann_index` so we can keep the original strategy structure.